client/data/blogging-prompt/use-blogging-prompt.ts

```diff
@@ -5,34 +5,45 @@ import wp from 'calypso/lib/wp';
 export interface BloggingPrompt {
 	id: string;
 	text: string;
+	answered: boolean;
+	answered_users_count: number;
+	answered_users_sample: [ string ];
 }
 
-const selectPrompt = ( response: any ): BloggingPrompt | null => {
-	const prompt = response && response.prompts && response.prompts[ 0 ];
-	if ( ! prompt ) {
+const selectPrompts = ( response: any ): [ BloggingPrompt ] | null => {
+	const prompts = response && response.prompts;
+	if ( ! prompts ) {
 		return null;
 	}
-	return {
-		id: prompt.id,
-		text: prompt.text,
-	};
+	return prompts.map(
```

Comment:
At this point, we're using most of the keys from the response. Maybe we don't need to map the values?

<img width="456" alt="image" src="https://user-images.githubusercontent.com/1699996/212784882-c3d87289-cf22-4af8-b488-c787e2661c61.png">